.github/workflows/build-test.yml

```diff
@@ -109,16 +109,10 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                image: [ 'nodejs-centos6-default', 'nodejs-centos7-node12' ]
+                image: [ 'nodejs-centos6-default', 'nodejs-centos7-node12', 'nodejs-centos7-node14' ]
```

Comment:
Do we still need centos6 here?

Reply:
I'm going to cleanup centos6 and node 12 builds in separate commits